packages/lesmis/src/components/FetchLoader.js

```diff
@@ -30,7 +31,7 @@ export const FetchLoader = ({ isLoading, isError, error, children, Loader }) =>
     );
   }
 
-  return children;
+  return <ErrorBoundary>{children}</ErrorBoundary>;
```

Comment:
This will show an error message instead of crashing the app if there are any errors rending in the children